packages/libs/src/sdk/services/Solana/programs/types.ts

```diff
@@ -39,7 +39,8 @@ export const BuildTransactionSchema = z
           )
           .default([])
       ])
-      .optional()
+      .optional(),
+    priorityPercentile: z.number().min(0).max(100).optional()
```

Comment:
Perhaps we should allow for higher than 100? I don't see a reason why it has to be capped at 100. I imagine there is some circumstance where we would like to send `largestMostRecentPriorityFee + 1`